fsharp-backend/testfiles/execution/language.tests

```diff
@@ -85,6 +85,49 @@ toString 5 = "5"
 (if blank then "" else "") = blank
 (if 5 then "correct" else "") = "correct"
 
+[tests.and]
```

Comment:
It might be worth adding some `"true"` and `1` args to some tests, though probably excessive